src/main/java/com/cloudbees/jenkins/plugins/docker_build_env/DockerBuildWrapper.java

```diff
@@ -129,6 +139,10 @@ public String getCommand() {
         return command;
     }
 
+    public String getProtectedEnvironmentVariables() {
```

Comment:
Add CheckForNull. Even if null is not being passed from UI, this field will be null when the build wrapper is being loaded from the disk for existing jobs. 🐛